### PR TITLE
initial implementation

### PR DESCRIPTION
**Pull request by mcalhoun**
## what
* Initial Implementation of the module

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,32 +70,63 @@ Security scanning is graciously provided by Bridgecrew. Bridgecrew is the leadin
...
+module "github_action_token_rotator" {
+    source = "cloudposse/github-action-token-rotator/aws"
+    # Cloud Posse recommends pinning every module to a specific version
+    # version = "x.x.x"
+    parameter_store_token_path       = "/github/runners/cloudposse/registrationToken"
+    parameter_store_private_key_path = "/github/runners/cloudposse/privateKey"
+    github_app_id                    = "111111"
+    github_app_installation_id       = "22222222"
+    github_org                       = "cloudposse"
+}```
```
Comment (Benbentwo) [inline README.md]: this needs to be split to a new line for formatting

File: README.md [hunk 2]
```diff
@@ -68,32 +70,63 @@ Security scanning is graciously provided by Bridgecrew. Bridgecrew is the leadin
...
+We highly recommend that in your code you pin the version to the exact version you are
+using so that your infrastructure remains stable, and update versions in a
+systematic way so that they do not catch you by surprise.
+
+Also, because of a bug in the Terraform registry ([hashicorp/terraform#21417](https://github.com/hashicorp/terraform/issues/21417)),
+the registry shows many of our inputs as required when in fact they are optional.
+The table below correctly indicates which inputs are required.
+
+
+usage: |-
```
Comment (Benbentwo) [inline README.md]:
```suggestion
usage:
```

not yaml :)

**Review by Benbentwo — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mcalhoun) [reply README.md]: f1df56ddf195651a4a3f80643b38faa7c5c65795

File: README.md (hunk 2 above)
Comment (mcalhoun) [reply README.md]: f1df56ddf195651a4a3f80643b38faa7c5c65795

**Review by Benbentwo — APPROVED** [review]
LGTM!

**Pull request merged**